docs/parameters.md

```diff
@@ -0,0 +1,150 @@
+# Installation Parameters
+
+This document covers installation parameters as presented to users, and includes how the puppet-based `foreman-installer` parameters will map to new parameters.
+
+## Design Principles
+
+* Using a single parameter when there are multiple variables that should have the same value
+* Aim for minimal configuration values for the user
+
+## Mapping
+
+Parameters are split into either mapped or unmapped groupings per category.
+
+Mapped: Parameters the new installer will have and are mapped to a parameter from the `foreman-installer`.
+Unmapped: Parameters from `foreman-installer` that appear in the official [foreman-documentation](https://github.com/theforeman/foreman-documentation) that are unmapped into the new installer.
+
+### Foreman
+
+These are parameters that are related to the install and operation of the Foreman server and plugins.
+
+#### Database
+
+There are multiple use cases from the users perspective that dictate what parameters need to be available for user input.
+
+ * The user has deployed an external database and needs to specify connection details such as host and port.
+ * The user wishes to manage passwords externally to the installer and needs to specify rather than use the default randomly generated password.
+ * The user has deployed an external database configured with TLS and needs to specify the CA certificate.
+ * The user has deployed an external database configured with TLS and certificate authentication.
```

Comment:
We don't support this today, and while we certainly could, I wonder if this is the right time to do?

Reply:
Fair. That makes life easier.

Reply:
Removed